pdp/static/js/vic_map.js

```diff
@@ -1,14 +1,30 @@
 /*jslint browser: true, devel: true */
 /*global $, jQuery, OpenLayers, pdp, Colorbar, BC3005_map_options, getBasicControls, getBoxLayer, getEditingToolbar, getHandNav, getBoxEditor, getBC3005Bounds_vic, getBC3005OsmBaseLayer, getOpacitySlider*/
 
+
+
 "use strict";
 
-function init_vic_map() {
+function init_vic_map(archive_portal) {
     var options, mapControls, selLayerName, selectionLayer, panelControls,
         defaults, map, params, datalayerName, cb, ncwms;
 
+	// Initial options vary by whether we're mapping the old or
+	// new dataset.
+	const vic_gen1_init = {
+		dataset:"5var_day_CCSM3_A1B_run1_19500101-20991231",
+		variable: "sm",
+		time:  "2000-01-01T00:00:00Z"
+		};
+	const vic_gen2_init = {
+		dataset: "BASEFLOW_day_VICGL_ACCESS1-0_rcp85_r1i1p1_19450101-20991231_columbia",
+		variable: "BASEFLOW",
+		time: "2000-01-01T00:00:00Z"
+	};
+	const init = archive_portal ? vic_gen1_init : vic_gen2_init;
+
     // Map Config
-    options = BC3005_map_options();
+    options = BC3005_map_options_vic(archive_portal);
```

Comment:
`BC3005_map_options_vic` doesn't use its argument (at least so it seems). If not, remove it here too?